modules/admin_manual/pages/enterprise/user_management/saml_2.0_sso.adoc

```diff
@@ -53,17 +55,17 @@ php apps/user_shibboleth/tools/adfs2fed.php \
 Next, you need to configure `shibd`.
 To do this, in `/etc/shibboleth/shibboleth2.xml`:
 
-=== Use the URL of the ownCloud instance as the `entityID` in the `ApplicationDefaults`, e.g.,
+=== Use the URL of the ownCloud instance as the `entityID` in the `ApplicationDefaults`
 
 [source,console]
 ....
 <ApplicationDefaults entityID="https://<owncloud server FQDN>/login/saml" REMOTE_USER="eppn upn">
 ....
 
-NOTE: `https://<owncloud server FQDN>/login/saml` is just an example.
+NOTE: `\https://<owncloud server FQDN>/login/saml` is just an example. +
```

Comment:
Can the `+` be removed, so that it's a naturally flowing paragraph, pls?

Reply:
No, If you render it you see the difference. Without the line break, it becomes hard to read, with the line break it is natural reading. If I make a line break, I do this by intention and have have checked the result by rendering it.